e2e/pages/manageCases/createCase/C100/childrenAndRespondents/childrenAndRespondents1Page.ts

```diff
@@ -0,0 +1,210 @@
+  childLiveWithThisPersonYes = "#buffChildAndRespondentRelations_0_childLivesWith_Yes",
+  childLiveWithThisPersonNo = "#buffChildAndRespondentRelations_0_childLivesWith_No",
+}
+
+enum nameSelectors {
+  nameSelector = ".case-field__label",
+}
+
+export class ChildrenAndRespondents1Page {
+  public static async childrenAndRespondents1Page(
+    page: Page,
+    accessibilityTest: boolean,
+    errorMessaging: boolean,
+    yesNoChildrenAndRespondents: boolean,
+  ): Promise<void> {
+    await this.checkPageLoads(page, accessibilityTest);
+    if (errorMessaging) {
+      await this.triggerErrorMessages(page);
+    }
+    await this.fillInFields(page, yesNoChildrenAndRespondents);
+  }
+
+  private static async checkPageLoads(
+    page: Page,
+    accessibilityTest: boolean,
+  ): Promise<void> {
+    await page.waitForSelector(
+      `${Selectors.h2}:text-is("${ChildrenAndRespondents1Content.h21}")`,
+    );
+    await Promise.all([
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.GovukHeadingL}:text-is("${ChildrenAndRespondents1Content.pageTitle}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.h2}:text-is("${ChildrenAndRespondents1Content.h22}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.h3}:text-is("${ChildrenAndRespondents1Content.h3}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.p}:text-is("${ChildrenAndRespondents1Content.p}")`,
+        1,
+      ),
+      Helpers.checkGroup(
+        page,
+        2,
+        ChildrenAndRespondents1Content,
+        "formLabel",
+        Selectors.GovukFormLabel,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${nameSelectors.nameSelector}:text-is("${ChildrenAndRespondents1Content.respondentsNameFormLabel}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${nameSelectors.nameSelector}:text-is("${ChildrenAndRespondents1Content.childNameFormLabel}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.GovukText16}:text-is("${ChildrenAndRespondents1Content.fullNameRespondent}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.GovukText16}:text-is("${ChildrenAndRespondents1Content.fullNameChild}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.GovukFormLabel}:text-is("${ChildrenAndRespondents1Content.formLabelYes}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.GovukFormLabel}:text-is("${ChildrenAndRespondents1Content.formLabelNo}")`,
+        1,
+      ),
+    ]);
+    if (accessibilityTest) {
+      // await AccessibilityTestHelper.run(page);
```

Comment:
Add the ticket number as a comment for why these a11y checks are commented please